array-manipulation-3/src/unique.js

```diff
@@ -29,3 +29,4 @@ function unique(array) {
   }
   return uniqueArr;
 }
+//
```

Comment:
Having a nested loop in this function should not be necessary. You should just loop over `array` and check if `uniqueArr` contains that value. If not, add that value to the array.